src/docx/assets/indywidualne/docProps/core.xml

```diff
@@ -1,11 +1,13 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
 <cp:coreProperties xmlns:cp="http://schemas.openxmlformats.org/package/2006/metadata/core-properties" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcmitype="http://purl.org/dc/dcmitype/" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <dcterms:created xsi:type="dcterms:W3CDTF">2019-07-14T15:37:29Z</dcterms:created>
-  <dc:creator/>
-  <dc:description/>
-  <dc:language>en-US</dc:language>
-  <cp:lastModifiedBy/>
-  <cp:revision>0</cp:revision>
-  <dc:subject/>
   <dc:title/>
+  <dc:subject/>
+  <dc:creator>python-docx</dc:creator>
```

Comment:
We should remember to remove that `python` description in the future.

Reply:
We will. It also does not properly render all fields (as I don't fully understand the logic behind it). That's just first iteration here. Will refine on it over time (creating also a lot of small tickets to be done in no-time).